src/ec2/commands.ts

```diff
@@ -30,3 +32,14 @@ export async function tryConnect(selection?: Ec2Selection): Promise<void> {
     const ec2Client = new Ec2ConnectionManager(selection.region)
     await ec2Client.attemptEc2Connection(selection)
 }
+
+export async function copyInstanceId(instanceId: string): Promise<void> {
```

Comment:
This is doing the same thing as [copyName](https://github.com/aws/aws-toolkit-vscode/blob/926fca0d358bd36c473febf182e9c30e4a2af198/src/awsexplorer/commands/copyName.ts#L14) and [copyArn](https://github.com/aws/aws-toolkit-vscode/blob/926fca0d358bd36c473febf182e9c30e4a2af198/src/awsexplorer/commands/copyArn.ts#L21).

This might be the right time to generalize them to a `src/awsexplorer/commands/copyText.ts` command. Then we only need test coverage for the generalized file. And we can delete `src/awsexplorer/commands/copyArn.ts` (all of its call sites would change to `await copyTextCommand(node, env, 'ARN')`).

Reply:
I gave the refactor a shot, but was unsure whether to include the instanceId as part of the same functionality. I figured it only makes sense if multiple services carry a general Id tag that could be helpful to copy.